chimg/chroot.py

```diff
@@ -271,7 +272,10 @@ def _files_install(self):
         """
         logger.info("Installing files ...")
         for f in self._ctx.conf["files"]:
-            self._file_install(f)
+            if f.get("content"):
+                self._file_install(f)
+            else:
+                self._local_file_upload(f)
```

Comment:
can we rename this to `_file_install_from_path()`. that would imo be more obvious. what do you think?